### DISCUSSION TITLE
From where I can learn Pixi.js To create creative web design Like Bruno Inbrizi

### DISCUSSION THREAD

**Discussion by whizzbbig** [pixijs/pixijs#7875] [Q&A]
Hi I always wanted to learn how i can create the same artwork i create in AE in web browser and can go more creative with them I have ideas and steps to approach those type of process but i am lacking the way and here i found Pixi.js can help me to implement those type of process I really like from people who are so pro and experienced with this library can share their story that how he/she worked on them and which course or youtube tutorial i can follow to get started with Pixi.js the right course and right tutorial by taking them i just straight dive in and start making projects rather getin distracted by more tuts :) thanks for your time

**Comment by miltoncandelero** [comment]
No idea who `Bruno Inbrizi` is 😅 

To begin you can try guides or the elementals, that is

Guides for more generic use: https://pixijs.io/guides/

Elementals is generic but videogame oriented: https://www.pixijselementals.com/#introduction

**Comment by ivanpopelyshev** [comment]
I know only this Bruno: https://twitter.com/bruno_simon

>  i can follow to get started with Pixi.js the right course and right tutorial by taking them i just straight dive in and start making projects rather getin distracted by more tuts

Yuri Artiukh has many has both threejs and pixi creative stuff, for example https://www.youtube.com/watch?v=czETLIAeAUU
